logs.hpp

```diff
@@ -292,16 +292,21 @@ namespace MTL_LOG_NAMESPACE
                 MTL_LOG_LOCK;
                 return MTL_LOG_NAMESPACE::Options::FORMAT;
             }
-#           ifdef _REENTRANT
-            static void bindThreadName(const std::thread::id& id, const std::string& name)
+
+            static void bindThreadName(const std::string& name)
             {
-                MTL_LOG_NAMESPACE::Options::THREAD_NAME.insert(std::make_pair(id, name));
+#               ifdef MTL_LOG_WITH_THREADS
+                MTL_LOG_LOCK;
+                MTL_LOG_NAMESPACE::Options::THREAD_NAME.insert(std::make_pair(std::this_thread::get_id(), name));
+#               endif
             }
-            static void unbidThreadName(const std::thread::id& id)
+            static void unbidThreadName()
```

Comment:
unbid is still here, and I be explicit about functions that doesn't take parameter (be consistent with the entire file) with void keyword.